include/GuiApplication.h

```diff
@@ -67,11 +67,17 @@ class LMMS_EXPORT GuiApplication : public QObject
 	AutomationEditorWindow* automationEditor() { return m_automationEditor; }
 	ControllerRackView* getControllerRackView() { return m_controllerRackView; }
 
+	void sendInterrupt(int signal);
```

Comment:
There's a slight bit of redundancy by calling it "sendInterrupt()` to then double-check that the code was `SIGINT`.  I'm fine genericizing and simplifying this further if needed.  I'd rather call this `sendSignal`, but in Qt-land, the word "signal" has its own meaning and I was for a loss of words.

Reply:
Resolving, since this code needs to change before merge.